tests/robot-tests/tests/admin_and_public/bau/footnotes.robot

```diff
@@ -29,26 +29,7 @@ Create new publication and release via API
 
 Upload subject
     [Tags]  HappyPath
-    user waits for page to finish loading
-    user navigates to release summary from admin dashboard  ${PUBLICATION_NAME}   Academic Year 2025/26 (not Live)
-    user clicks link  Data and files
-    user waits until page contains element  id:dataFileUploadForm-subjectTitle
-    user enters text into element  id:dataFileUploadForm-subjectTitle   ${SUBJECT_NAME}
-    user chooses file   id:dataFileUploadForm-dataFile       ${FILES_DIR}${DATA_FILE_NAME}.csv
-    user chooses file   id:dataFileUploadForm-metadataFile   ${FILES_DIR}${DATA_FILE_NAME}.meta.csv
-    user clicks button  Upload data files
-
-    user waits until h2 is visible  Uploaded data files
-    user waits until page contains accordion section   ${SUBJECT_NAME}
-    user opens accordion section   ${SUBJECT_NAME}
-
-    ${section}=  user gets accordion section content element  ${SUBJECT_NAME}
-    user checks headed table body row contains  Subject title    ${SUBJECT_NAME}  ${section}
-    user checks headed table body row contains  Data file        dates.csv  ${section}
-    user checks headed table body row contains  Metadata file    dates.meta.csv  ${section}
-    user checks headed table body row contains  Number of rows   119  ${section}
-    user checks headed table body row contains  Data file size   17 Kb  ${section}
-    user checks headed table body row contains  Status           Complete  ${section}  360
+    user uploads subject   ${SUBJECT_NAME}  ${DATA_FILE_NAME}.csv  ${DATA_FILE_NAME}.meta.csv
```

Comment:
I think you need `user navigates to release summary from admin dashboard ${PUBLICATION_NAME} Academic Year 2025/26 (not Live)` still before you can call `user uploads subject`